it/uk/gov/hmrc/partnershipidentificationfrontend/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.partnershipidentificationfrontend.controllers
+
+import play.api.libs.ws.WSResponse
+import play.api.test.Helpers._
+import uk.gov.hmrc.partnershipidentificationfrontend.assets.TestConstants._
+import uk.gov.hmrc.partnershipidentificationfrontend.stubs.{AuthStub, PartnershipIdentificationStub}
+import uk.gov.hmrc.partnershipidentificationfrontend.utils.ComponentSpecHelper
+import uk.gov.hmrc.partnershipidentificationfrontend.views.CheckYourAnswersViewTests
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class CheckYourAnswersControllerISpec extends ComponentSpecHelper
+  with CheckYourAnswersViewTests
+  with PartnershipIdentificationStub
+  with AuthStub {
+
+  override def afterEach(): Unit = {
+    super.afterEach()
+    journeyConfigRepository.drop
+  }
```

Comment:
This isn't needed, the ComponentSpecHelper has a beforeEach method that does this.